packages/discord.js/src/structures/GuildMember.js

```diff
@@ -63,7 +64,7 @@ class GuildMember extends Base {
        * The user that this guild member instance represents
        * @type {?User}
```

Comment:
When will this be nullable? Even if data.user would be undefined it will still have an (albeit invalid) user class

Reply:
it won't set the prop if data.user is undefined, there's a lot of places where we use ?Type to indicate undefined instead of explicitly making it null still.